templates/company/view_all/view.html.tx

```diff
@@ -2,65 +2,76 @@
+          <span class="govuk-visually-hidden">link opens in new tab/window</span></a>
+        <div id="status-widget" class="widget-footer hidden">
+          <div id="status">
+            <div id="process-spinner" class="process-spinner"></div>
+              <span id="process-status" class="process-status">Status: in-progress</span>
+            </div>
+            <input type="hidden" id="document-data"
+              data-resource-url="/company-number/<%$company.company_number%>"
+              data-content-type="text/html" data-document-type="text/html">
+            </input>
           </div>
-          <input type="hidden" id="document-data"
-            data-resource-url="/company-number/<%$company.company_number%>"
-            data-content-type="text/html" data-document-type="text/html">
-          </input>
         </div>
       </div>
-    </div>
-    <div id="download-button"></div>
-    % } else {
-    <a class="govuk-button secondary-button piwik-event" data-event-id="View company information snapshot" href="<% $c.url_for('user_sign_in') %>">View company information snapshot
-        <span class="govuk-visually-hidden">link opens in new tab/window</span></a>
+      <div id="download-button"></div>
+      % } else {
+      <a class="govuk-button secondary-button piwik-event" data-event-id="View company information snapshot" href="<% $c.url_for('user_sign_in') %>">View company information snapshot
+          <span class="govuk-visually-hidden">link opens in new tab/window</span></a>
     % }
-
-  % } else {
-    <article class="text">
-      <header>
-        <h1 id="page-not-found-header" class="heading-xlarge">
-          This page cannot be found
-        </h1>
-      </header>
-    <p>Check that you have entered the correct web address or try using the search.</p>
-    <p>If you still can't find what you're looking for email <a href="mailto:enquiries@companieshouse.gov.uk">enquiries@companieshouse.gov.uk</a></p>
-  </article>
   % }
+
+  % if $show_orders {
+    <h2 id="order-a-certificate-header" class="heading-large heading-with-border">
+      Order a certificate
+    </h2>
+    <details class="govuk-details" role="group">
+      <summary class="govuk-details__summary" role="button" aria-controls="details-content" aria-expanded="true">
+        <span class="govuk-details__summary-text">
+          What you can order
+        </span>
+      </summary>
+      <div class="govuk-details__text" id="details-content" aria-hidden="false">
+        <p>You can:</p>
+        <ul class="govuk-list govuk-list--bullet">
+          <li>order a certificate of incorporation</li>
+          <li>include good standing information</li>
+          <li>pay by debit or credit card</li>
+        </ul>
+      </div>
+    </details>
+    <a class="govuk-button secondary-button piwik-event" data-event-id="Order a certificate" href="<% $c.url_for('/orderable/certificates') %>">Order certificate</a>
```

Comment:
piwik-event tag is unnecessary as there have been changes to CDN

Reply:
Removed piwik event tag